dotnet-client-libraries/BenchmarkDotNetWithUnitTests/BenchmarkDotNetWithUnitTests/StringConcatBenchmarks.cs

```diff
@@ -0,0 +1,56 @@
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Order;
+
+namespace BenchmarkDotNetWithUnitTests;
+
+[MemoryDiagnoser]
+[Orderer(SummaryOrderPolicy.FastestToSlowest)]
+[RankColumn]
+public class StringConcatBenchmarks
+{
+    private string? _firstName;
```

Comment:
These could just be constant strings, since they are not being changed anywhere in the code.